Userland/Tests/Kernel/elf-execve-mmap-race.cpp

```diff
@@ -152,8 +152,8 @@ int main()
         try_again:
             printf("exec\n");
             if (execl(path, "x", nullptr) < 0) {
+                goto try_again;
             }
-            goto try_again;
```

Comment:
If `execl()` fails, we want it to try again, if it doesn't, then this `goto` can never be reached (short of a bug in `execve`).
this whole `if` is rather meaningless here, I think.

I suppose it's okay to be explicit about the jump though.

Reply:
Removed the `if` completely.